demo/tiles/circles-page.html

```diff
@@ -54,35 +54,40 @@
     properties: {
 
       animationConfig: {
-        value: function() {
-          var circles = Polymer.dom(this.root).querySelectorAll('.circle');
-          var circlesArray = Array.prototype.slice.call(circles);
-          return {
-            'entry': [{
-              name: 'cascaded-animation',
-              animation: 'scale-up-animation',
-              nodes: circlesArray
-            }],
-
-            'exit': [{
-              name: 'hero-animation',
-              id: 'hero',
-              fromPage: this
-            }, {
-              name: 'cascaded-animation',
-              animation: 'scale-down-animation'
-            }]
-          };
-        }
+        type: Object
       }
     },
 
     listeners: {
       'click': '_onClick'
     },
 
+    attached: function() {
+      var circles = Polymer.dom(this.root).querySelectorAll('.circle');
+      var circlesArray = Array.prototype.slice.call(circles);
+      this.animationConfig = {
+        'entry': [{
+          name: 'cascaded-animation',
+          animation: 'scale-up-animation',
+          nodes: circlesArray
+        }],
+
+        'exit': [{
+          name: 'hero-animation',
+          id: 'hero',
+          fromPage: this
+        }, {
+          name: 'cascaded-animation',
+          animation: 'scale-down-animation'
+        }]
+      };
+    },
+
     _onClick: function(event) {
       var target = Polymer.dom(event).rootTarget;
+      if (!target) {
```

Comment:
This condition shouldn't be needed as `Polymer.dom(event).rootTarget` is already implemented as `event.composedPath()[0]`